applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -212,6 +212,83 @@ Example
    #XSLEEP: (1,2)
    OK
 
+Power off #XSHUTDOWN
+====================
+
+The ``#XSHUTDOWN`` command makes the nRF9160 SiP enter SYSTEM OFF mode, which is the deepest power saving mode.
+
+Set command
+-----------
+
+The set command makes the nRF9160 SiP enter SYSTEM OFF mode.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSHUTDOWN
+
+.. note::
+
+   * In this case the nRF9160 SiP cannot be woken up using the :ref:`CONFIG_SLM_WAKEUP_PIN <CONFIG_SLM_WAKEUP_PIN>`..
```

Comment:
```suggestion
   In this case the nRF9160 SiP cannot be woken up using the :ref:`CONFIG_SLM_WAKEUP_PIN <CONFIG_SLM_WAKEUP_PIN>`..
```
Don't need a bullet since it's the only thing inside the note.